tests/MapTest.php

```diff
@@ -2659,6 +2659,33 @@ public function testZip()
 
 		$this->assertEquals( $expected, $m->zip( $en, $es )->toArray() );
 	}
+
+	public function testIfThen()
+	{
+		Map::from( ['a' => 1, 'b' => 0] )->if(
+			fn(Map $map) => $map->has('a'),
+			function(Map $_) { $this->assertTrue(true); },
+			function(Map $_) { $this->assertTrue(false);}
+		);
+	}
+
+	public function testIfElse()
+	{
+		Map::from( ['a' => 1, 'b' => 0] )->if(
+			fn(Map $map) => $map->has('c'),
+			function(Map $_) { $this->assertTrue(false); },
+			function(Map $_) { $this->assertTrue(true);}
+		);
+	}
+
+	public function testIfBlankElse()
+	{
+		Map::from( ['a' => 1, 'b' => 0] )->if(
+			fn(Map $map) => $map->has('c'),
+			function(Map $_) { $this->assertTrue(false); }
+		);
+		$this->assertTrue(true);
+	}
```

Comment:
Can you please move this up between testHas*() and testIn*()?

Reply:
Fixed in 3ccaeb0ad8a3ba8f79258d8908ed6bab774a3a41

Reply:
...and 1133f756987a045d62dd1194dd36fba0fb5d9ab2 (sorts alphabetically the three added tests)